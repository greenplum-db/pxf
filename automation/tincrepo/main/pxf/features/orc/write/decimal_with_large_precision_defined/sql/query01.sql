-- @description query01 for writing ORC decimals with large precision defined in the table DDL
INSERT INTO orc_decimals_with_large_precision_writable VALUES (1, 1234567890123456789012345678901234567890.12345);
