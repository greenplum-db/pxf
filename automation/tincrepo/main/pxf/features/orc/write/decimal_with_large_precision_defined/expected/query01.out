-- start_ignore
-- end_ignore
-- @description query01 for writing ORC decimals with large precision defined in the table DDL
INSERT INTO orc_decimals_with_large_precision_writable VALUES (1, 1234567890123456789012345678901234567890.12345);
ERROR: PXF server error : Column c_numeric is defined as NUMERIC with precision 90 which exceeds the maximum supported precision 38.
